docs/blog/posts/2023-08-08-using_twarc.md

```diff
@@ -128,14 +130,14 @@ The Academic Twitter API allows you to search all tweets, not just the last 7 da
 using `client.search_all()` function from `twarc` package. To get all tweets (without comments) for our query,
  run the following:
 
-```python
+```bash linenums="1" hl_lines="4-4" title="python"
```

Comment:
and here

Reply:
I think you're looking at outdated file but I changed all python to py